js/ui/html_editor/ui.html_editor.js

```diff
@@ -545,23 +555,23 @@ const HtmlEditor = Editor.inherit({
     },
 
     /**
-    * @name dxHtmlEditorMethods.registerModules
-    * @publicName registerModules(modules)
+    * @name dxHtmlEditorMethods.register
+    * @publicName register(modules)
```

Comment:
This and `get` methods also work with formats, so "modules" is not a suitable name for the parameter. My best suggestion so far is `components`. Maybe, @TikhonovaTatyana can come up with something better?

Reply:
In Quill, the [section](https://quilljs.com/docs/api/) with similar methods is called "Extension". Maybe we can also take this term? 
I remember that we discussed "extension" but do not remember why it was not accepted.

Reply:
The section is called so because methods described in it extend the built-in functionality. Essentially, these are extension methods. Quill do not use the word "extension" when referring to modules or formats.

Reply:
ok, let's use `components`

Reply:
Done